recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/mbop/MBOPService.java

```diff
@@ -43,6 +43,7 @@ MBOPUsers getUsersByOrgId(
         @RestQuery("limit")                             int limit,
         @RestQuery("offset")                            int offset,
         @RestQuery("include_permissions")               boolean includePermissions,
-        @RestQuery("status")                            String status
+        @RestQuery("status")                            String status,
+        @RestQuery("skipTotalCount")                    boolean skipTotalCount
```

Comment:
IT's related to [RHCLOUD-32276](https://issues.redhat.com/browse/RHCLOUD-32276): it's about skiping an extra count request to retreive total number of users.
We don't need it in our use case, and skiping it will improve response time.

Reply:
Sounds good! Thanks for the answer.